### PR TITLE
Show error when u2f factor is selected on mobile device.

### PR DESCRIPTION
**Pull request by alexeystadnikov-okta**
Resolves OKTA-111230

screen shot 2018-04-03 at 4 57 53 pm

i18n PR: https://github.com/okta/npm/pull/1034

@mauriciocastillosilva-okta 
Please, review.

### CODE REVIEW

File: src/VerifyU2FController.js [hunk 1]
```diff
@@ -99,14 +99,21 @@ function (Okta, FormController, FormType, FooterSignout, Q, HtmlErrorMessageView
...
+          var isOneFactor = this.options.appState.get('factors').length === 1;
 
-          if (BrowserFeatures.isFirefox()) {
-            errorMessageKey = 'u2f.error.noFirefoxExtension';
+          if (isMobileDevice) {
+            errorMessageKey = isOneFactor ? 'u2f.error.mobileDevice.oneFactor' : 'u2f.error.mobileDevice';
           }
-          else if (this.options.appState.get('factors').length === 1) {
-            errorMessageKey = 'u2f.error.notSupportedBrowser.oneFactor';
+          else {
```
Comment (mauriciocastillosilva-okta) [inline src/VerifyU2FController.js]: nit:
```
else if (BrowserFeatures.isFirefox()) {
}
else if (isOneFactor) {
}
```

File: packages/@okta/i18n/package.json [hunk 2]
```diff
@@ -1,83 +1,109 @@
...
-  "_spec": "@okta/i18n@42.3.0",
-  "_where": "/Users/santhosh/repos/okta/santhoshbalakrishnan/okta-signin-widget",
+  "_resolved": "https://artifacts.aue1d.saasure.com:443/artifactory/api/npm/npm-okta/@okta/i18n/-/@okta/i18n-44.2.0-beta.1.tgz",
+  "_shasum": "c73866b741dd5734a3a224d77f56a03ddd89c669",
+  "_spec": "@okta/i18n@44.2.0-beta.1",
+  "_where": "/Users/astadnikov/okta/okta-signin-widget",
   "bin": {
     "i18n": "dist/bin/i18n.js"
   },
+  "bundleDependencies": false,
```
Comment (mauriciocastillosilva-okta) [inline packages/@okta/i18n/package.json]: what are these new values? Do you happen to know by any chance?
Taling about `bundleDependencies` and `deprecated`

Comment (alexeystadnikov-okta) [reply packages/@okta/i18n/package.json]: They were added by npm, but seems are optional.
Removed them.

**Comment by mauriciocastillosilva-okta** [comment]
Changes were merged in https://github.com/okta/okta-signin-widget/pull/460

**Pull request closed without merging**